_posts/2020-09-15-summer-sprint-2020.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: GeoBlacklight 3.0 is Here!
+date: 2020-09-15 12:00:00
+categories: blog
+author: Karen Majewicz
+snippet: 'The GeoBlacklight Summer Community Sprint Activities## 2019 Winter Codesprint'
```

Comment:
Should this be updated? I see 2019 Winter

Reply:
deleted in last commit